server/src/main/java/org/elasticsearch/snapshots/RepositoryIntegrityHealthIndicatorService.java

```diff
@@ -46,19 +55,42 @@ public class RepositoryIntegrityHealthIndicatorService implements HealthIndicato
 
     public static final String HELP_URL = "https://ela.st/fix-repository-integrity";
```

Comment:
I followed this link and currently it points to a troubleshooting page that has the title `Multiple Elasticsearch deployments are writing to the same snapshot repository`. I think this is way too specific and we might need another help page for the new cases. What do you think?

Reply:
See [9d2a51e](https://github.com/elastic/elasticsearch/pull/104614/commits/9d2a51e7075bbf02be99521048f7ccc4b64f9475).